batcher.js

```diff
@@ -13,19 +13,19 @@ if (process.argv.length < 4) {
 
 var command = process.argv[2];
 var query = process.argv[3];
-var data = parseData(process.argv[4]);
+var searchParams = sToO(process.argv[4]);
+var data = sToO(process.argv[5]);
 
-function parseData(data) {
-    // FIXME: not particularly typesafe.
-    try {
-        return JSON.parse(data)
-    } catch(e) {
-        return undefined;
+function sToO(s) {
+    // Wow!
```

Comment:
The reason for this is so the commands look like:
`batcher patchMetadata "credit:'Getty Images'" "{ free: true }" "{ credit: 'Corbis' }"`